kuma/static/js/wiki.js

```diff
@@ -59,20 +49,6 @@
         });
     });
 
-    // Utility method for the togglers
-    function setupTogglers($elements) {
-        $elements.each(function() {
-            var $li = $(this);
-            var $sublist = $li.find('> ul, > ol');
-
-            if ($sublist.length) {
-                $li.addClass('toggleable closed');
-                $li.find('> a').addClass('toggler').prepend('<i aria-hidden="true" class="icon-caret-up"></i>');
```

Comment:
Here's the before and after on [Web/Events/addstream](https://developer.mozilla.org/en-US/docs/Web/Events/addstream), with ``{{EventRef}}``:

<img width="799" alt="pr 4957 - no indicator" src="https://user-images.githubusercontent.com/286017/45110003-b4709f80-b106-11e8-89a5-324100cbf060.png">

I think the caret indicator helps distinguish between links like "Events" and menus that open like "WebRTC Events". Is there a way to indicate this without the FontAwesome ``icon-caret-*``?

Reply:
Hmm it may just be {{EventRef}}. Here's [Web/API/Animation/id](https://developer.mozilla.org/en-US/docs/Web/API/Animation/id) with ``{{APIRef("Web Animations")}}``, before and after.

![pr 4957 - indicator](https://user-images.githubusercontent.com/286017/45121192-f1985a00-b125-11e8-8c01-541cce09ab19.png)

The weight of the section names are a little lighter, but it is functionally the same, with the indicator.

This includes the change from PR #4963, which allows the ``open`` attribute on ``details`` elements.  We'll need that as well, with a merge and rebase or you can cherry-pick it onto this branch.

I've opened [bug 1488884](https://bugzilla.mozilla.org/show_bug.cgi?id=1488884), since Kuma changes require a related bug. I can get the bug number into commit messages when we're ready to merge.

Reply:
Yeah...that surprised me. The little twistie thing should be free now that we use details/summary.

Reply:
So turns out I just missed that file. It _should_ be fixed in the Kumascript PR (https://github.com/mdn/kumascript/pull/789/commits/00fa08a97f78667126b317c47130bf40993d37e7)

Is there a secret to being able to test that page locally? Mine 404's.

Reply:
@tkadlec If you get a 404 it generally means you do not have that page locally and need to manually create it.

Reply:
The short version:

```
$ make bash # To start a shell in the web container
$ ./manage.py scrape_links https://developer.mozilla.org/en-US/docs/Web/API/Animation/id
```

See https://kuma.readthedocs.io/en/latest/data.html for all the details.

Reply:
Oh. my. goodness. I had no idea you could do it like this. That is going to save me a ton of time. Thanks @jwhitlock